pandas/tests/extension/test_common.py

```diff
@@ -79,3 +133,20 @@ def test_astype_no_copy():
 def test_is_extension_array_dtype(dtype):
     assert isinstance(dtype, dtypes.ExtensionDtype)
     assert is_extension_array_dtype(dtype)
+
+
+@pytest.mark.parametrize("na_value", [np.nan, pd.NA, None])
+def test_empty_series_construction(na_value):
+    class TempDType(DummyDtype):
+        @classmethod
+        def construct_array_type(cls):
+            return TempArray
+
+    TempDType.na_value = na_value
```

Comment:
could use monkeymatch.setitem on a single Dtype instead of creating new ones?  would that allow for any simplification?

Reply:
Is this referring to the pytest monkeypatch fixture?  I haven't used it before, so I'll get my head around it and see if I can answer your question.

Reply:
@jbrockmendel I've had a go in the latest commit.  It seems to work although docstring/typing check failed.

Reply:
that looks better, thanks

> although docstring/typing check failed.

that's probably the same one that is failing on main, i think a fix is coming soonish.  you can ignore that for now.